internal/distribute/rest/distribute.go

```diff
@@ -93,7 +93,10 @@ type Distributor struct {
 func (d *Distributor) DistributeOnce(ctx context.Context) error {
 	numErrs := 0
 	numLogs := 0
-	for log := range d.logConfig.Logs() {
+	for log, err := range d.logConfig.Logs(ctx) {
+		if err != nil {
+			return fmt.Errorf("failed to get logs iterator: %v", err)
```

Comment:
```suggestion
			return fmt.Errorf("failed to enumerate logs: %v", err)
```

The previous message was slightly misleading.

Reply:
Ta, fixed a similar one in `feedwitness` too